tests/ert_tests/shared/test_libres_facade_summarydata.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from unittest import TestCase
+from unittest.mock import patch
+import pytest
+
+from utils import SOURCE_DIR
+from ert_utils import tmpdir
+import pandas as pd
+
+from ert_shared.libres_facade import LibresFacade
+from res.enkf import EnKFMain, ResConfig
+
+from res.enkf.export import SummaryCollector
+
+_orig_loader = SummaryCollector.loadAllSummaryData
+
+
+def _add_duplicate_row(*args, **kwargs):
+    df = _orig_loader(*args, **kwargs)
+    # Append copy of last date to each realization
+    idx = pd.MultiIndex.from_tuples(
+        [(i, df.loc[i].iloc[-1].name) for i in df.index.levels[0]]
+    )
+    df_new = pd.DataFrame(0, index=idx, columns=df.columns)
+    return pd.concat([df, df_new]).sort_index()
+
+
+class LibresFacadeSummarydataTest(TestCase):
+    # TODO: same as in test_libres_facade.py - perhaps move into utils?
```

Comment:
TODO still hanging around, do you still need to to move to utils or was just forgotten here ?